crates/fluvio-cli/src/connector/mod.rs

```diff
@@ -112,20 +149,95 @@ impl ConnectorConfig {
 
 impl From<ConnectorConfig> for ManagedConnectorSpec {
     fn from(config: ConnectorConfig) -> ManagedConnectorSpec {
+        let mut parameters = BTreeMap::new();
+        for (key, value) in config.parameters.iter() {
+            parameters.insert(key.clone(), value.context.clone());
+        }
+
+        // Producer arguments are prefixed with `producer`
+        if let Some(producer) = config.producer {
+            if let Some(linger) = producer.linger {
+                let linger = humantime::format_duration(linger).to_string();
+                parameters.insert("producer-linger".to_string(), vec![linger]);
+            }
+            if let Some(compression) = producer.compression {
+                let compression = format!("{:?}", compression);
+                parameters.insert("producer-compression".to_string(), vec![compression]);
+            }
+            if let Some(batch_size) = producer.batch_size {
+                let batch_size = format!("{}", batch_size);
```

Comment:
Since we're adding units to linger, can we do the same w/ batch size? In the fluvio CLI (maybe currently only in `topic`), this is done w/ [ByteSize](https://crates.io/crates/bytesize).

It is not obvious from the key name what units this is in, and it would be reasonable for a user to assume bytes or kilobytes.

Reply:
Hmm. The `ByteSize` docs don't really show how one can use `1GB` is a param in json/serde.

Reply:
Hm. Yeah, docs aren't as clear as they could be.

I had to look at the [repo's code](https://github.com/hyunsik/bytesize/blob/master/src/parse.rs), but it seems like you can use `parse` with a type.

```
      fn parse(s: &str) -> u64 {
          s.parse::<ByteSize>().unwrap().0
      }
```

And at least the way we using in in `fluvio topic create`,  the  [`.as_u64()`](https://docs.rs/bytesize/1.1.0/bytesize/struct.ByteSize.html#method.as_u64) method returns the `ByteSize` value in bytes.